cli/mfg.c

```diff
@@ -786,6 +788,191 @@ static int security_config_set(int argc, char **argv)
 	return 0;
 }
 
+#if HAVE_LIBCRYPTO
+static int load_sig_from_file(FILE *sig_fimg, uint8_t *sig)
+{
+	ssize_t rlen;
+
+	rlen = fread(sig, 1, SWITCHTEC_SIG_LEN, sig_fimg);
+
+	if (rlen < SWITCHTEC_SIG_LEN)
+		return -1;
+
+	return 0;
+}
+
+static int kmsk_already_exist(uint8_t *kmsk,
+			      struct switchtec_security_cfg_stat *state)
```

Comment:
This might be too late.... but 'stat' is a horrible abbreviation for 'state' because it can be confused with "statistics".... maybe you can clean that up at some point.

Reply:
OK will update later.